style.css

```diff
@@ -0,0 +1,423 @@
+@font-face {
+  font-family: Fraunces-font;
+  src: url(./assets/Fraunces-VariableFont_SOFT\,WONK\,opsz\,wght.ttf);
+}
+
+@font-face {
+  font-family: Fraunces-font-bold;
+  src: url(./assets/static/Fraunces_72pt\,Soft-Bold.ttf);
+}
+
+footer {
+  color: gray;
+}
+
+.date {
+  color: white;
+}
+
+.container {
+  width: 100%;
+}
+
+.one {
+  background-color: #f72210;
+  width: 100%;
+}
+
+.btn-warning {
+  font-size: 6px;
+  border: 1px solid rgba(240, 173, 78, 0.75);
+  border-radius: 40px;
+  background: yellow;
+  color: black;
+  font-weight: bold;
+}
+
+.nav-login-text {
+  color: white;
+  font-size: 1.3rem;
+}
+
+.nav-subscribe-button {
+  font-size: 0.8rem;
+}
+
+.two {
+  font-weight: bold;
+}
+
+.gray {
+  color: gray;
+  font-size: 11px;
+}
+
+.images-float-left {
+  float: left;
+}
+
+.images-float-right {
+  float: right;
+}
+
+.search {
+  font-weight: lighter;
+}
+
+.news {
+  color: white;
+  font-size: 1.8rem;
+  font-weight: bolder;
+  font-family: Fraunces-font-bold, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+.news_red {
+  color: #f72210;
+}
+
+.big {
+  font-size: 2.5rem;
+  font-family: Fraunces-font-bold, 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-weight: bolder;
+}
+
+.three {
+  font-size: 0.9rem;
+  font-style: bolder;
+  color: gray;
+}
+
+.footer-copyright {
+  font-size: 0.8rem;
+}
+
+.edition {
+  font-weight: bolder;
+}
+
+.polaska,
+.pakistan,
+.china,
+.romania,
+.usa,
+.edition {
+  font-size: 0.8rem;
+}
+
+.four {
+  display: flex;
+  align-items: center;
+}
+
+.six {
+  font-size: 0.8rem;
+}
+
+.footer-links {
+  /* width: 15%;
+  display: inline-block; */
+}
+
+.break::after {
+  content: "\a\a";
+  white-space: pre;
+}
+
+.debra {
+  position: relative;
+}
+
+.tdebate {
+  margin-top: 20px;
+  border-bottom: 2px solid gray;
+}
+
+.pic_text {
+  margin-top: 15px;
+  border-bottom: 6px solid gray;
+  padding-bottom: 120px;
+  margin-bottom: 15px;
+  position: relative;
+  height: 110px;
+}
+
+.gray-text {
+  color: gray;
+  font-size: 11px;
+}
+
+.by {
+  margin-left: 63px;
+  margin-bottom: 6px;
+  font-size: 11px;
+}
+
+.dot {
+  height: 25px;
+  width: 25px;
+  background-color: black;
+  border-radius: 50%;
+  position: absolute;
+  bottom: -16px;
+  left: 45%;
+  font-size: 13px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  padding: 13px;
+}
+
+.opinion {
+  border-bottom: 2px solid gray;
+}
+
+.square {
+  position: absolute;
+  background-color: white;
+  width: 30%;
+  bottom: 0;
+  display: flex;
+  justify-content: center;
+  color: red;

```

Comment:
- [ ] Use bootstrap classes for position, background-color, display, justify-content, and color